github/repos.go

```diff
@@ -1037,3 +1037,38 @@ func (s *RepositoriesService) ReplaceAllTopics(ctx context.Context, owner, repo
 
 	return t.Names, resp, nil
 }
+
+// TransferRequest represents a request to transfer a repository.
+type TransferRequest struct {
+	NewOwner string `json:"new_owner"`
+	TeamID   []int  `json:"team_id,omitempty"`
```

Comment:
I just noticed now, I think `json:"team_id,omitempty"` should've been `json:"team_ids,omitempty"`, isn't that right?

From https://developer.github.com/v3/repos/#transfer-a-repository:

![image](https://user-images.githubusercontent.com/1924134/42599032-fb19cd96-852b-11e8-84e7-d2def7b17aca.png)

(FWIW, https://developer.github.com/v3/orgs/members/#create-organization-invitation also uses `team_ids`.)

Reply:
Good catch, @shurcooL!

Reply:
Thank you, @juliaferraioli! No rush... this is not urgent.